dashboard/config/routes.rb

```diff
@@ -774,7 +774,7 @@
 
   post '/i18n/track_string_usage', action: :track_string_usage, controller: :i18n
 
-  get 'java_ide', to: 'java_ide#index'
+  get 'idelab', to: 'idelab#index'
```

Comment:
Do we still want the url to say "java_ide?" (cc: @moneppo)

Reply:
i.e. then in the future we might also have a route for "python_ide"

Reply:
fwiw I think we can get rid of this url completely once we are integrated with code studio

Reply:
👍